Framework/PythonInterface/mantid/api/src/Exports/ITableWorkspace.cpp

```diff
@@ -720,6 +839,9 @@ void export_ITableWorkspace() {
 
       .def("column", &column, (arg("self"), arg("column")), "Return all values of a specific column as a list.")
 
+      .def("columnArray", &columnArray, (arg("self"), arg("column")),
+           "Return all values of a specific column as a numpy array.")
```

Comment:
```suggestion
           "Return all values of a specific column (either index or name) as a numpy array.")
```
I think this should say how to pick a column, because otherwise the tooltip in mantid just says `(object)column`